docs/upgrade-notes/upgrade-rds.md

```diff
@@ -122,6 +122,16 @@ Please read this entire section thoroughly before proceeding to run the second d
 
 :::
 
+:::note
+
+The data migration Lambda described here uses Cumulus' Granule `upsert` logic to write granules from DynamoDB to PostgreSQL. This is particularly notable because granules with a `running` or `queued` status will only migrate a subset of their fields. See the Granule `upsert` logic [here](https://github.com/nasa/cumulus/blob/release-15.0.x/packages/db/src/models/granule.ts#L128).
```

Comment:
Nit:  Probably should explicitly note this is referring to `data-migration-2` somehow

Reply:
https://github.com/nasa/cumulus/pull/3385/commits/f1c6413c52eba085f84023a8e5ccc5b23278e3f9